test/tests_for_sdkmr/ee2_SDKMethodRunner_EE2Logs_test.py

```diff
@@ -35,7 +36,7 @@ def setUpClass(cls):
         cls.token = "token"
 
         cls.method_runner = SDKMethodRunner(
-            cls.cfg, user_id=cls.user_id, token=cls.token
+            cls.cfg, UserClientSet(cls.cfg, cls.user_id, cls.token)
```

Comment:
The codebase usually does the following

def foo(a):
is called 
`foo(a)`

`def foo(a,b):`
is called
`foo(a=1,b=1)`

Reply:
So we should probably have some consistency in the codebase.

Reply:
I thought that you used keyword arguments for optional arguments and positional arguments for required arguments? E.g. the cfg is required, so it's not a keyword in either version, and UCS is required in the new version so it's not a keyword arg either.

Reply:
@MrCreosote If you look at the codebase, that is the pattern that is often used, so we should stick to one way and do it that way, in order to maintain consistency.

Reply:
I don't think I understand the pattern. Is it 1 arg = no keyword arguments, > 1 arg = all keyword arguments, regardless of the function signature?

Reply:
Yea that's probably bad come to think of it.  The issue is calling functions with a large function signature with a bunch of positional arguments makes it hard to remember what is going on. Maybe we need to create an issue to address how we deal with that stylistically for the ee2 repo and save it to the contributing.md file

Reply:
Feel free to correct / elaborate: https://github.com/kbase/execution_engine2/issues/303

Reply:
perfect!